2019-03-Autumn/2_string_manipulation/palindrome_permutation/Palindrome_Test.java

```diff
@@ -0,0 +1,182 @@
+
+public class Palindrome_Test {
+    /**
+     * Solution #1
+     */
+    static boolean isPermutationOfPalindrome(String phrase) {
+        int[] table = buildCharFrequencyTable(phrase);
+        return checkMaxOneOdd(table);
+    }
```

Comment:
This is a good solution, but I think it might be less confusing for folks if we include an example of accomplishing the same thing without having to implement our own hash table.